tests/Integration/Classes/controller/AdminControllerTest.php

```diff
@@ -293,8 +294,11 @@ private function getMockContainerBuilder(): ContainerBuilder
                 if ($param === 'PrestaShop\PrestaShop\Core\Image\AvifExtensionChecker') {
                     return $this->getMockedAvifExtensionChecker();
                 }
-                if ($param === 'prestashop.core.admin.feature_flag.repository' || $param === FeatureFlagRepository::class) {
-                    return $this->getMockedFeatureFlagRepository();
+                if (
+                    $param === 'PrestaShop\PrestaShop\Core\FeatureFlag\FeatureFlagProviderInterface' ||
```

Comment:
I think both of the conditions are equals

Reply:
Totally! I will change that.